app/src/test/java/org/mozilla/fenix/search/toolbar/SearchSelectorToolbarActionTest.kt

```diff
@@ -65,7 +83,6 @@ class SearchSelectorToolbarActionTest {
             SearchSelectorToolbarAction(
                 store = store,
                 menu = menu,
-                viewLifecycleOwner = lifecycleOwner
```

Comment:
I think you could also remove the mock from declarations and also the `MockedLifecycleOwner` class.